test/extensions/filters/http/cache/simple_http_cache/simple_http_cache_test.cc

```diff
@@ -228,7 +249,7 @@ TEST(Registration, GetFactory) {
 
 TEST_F(SimpleHttpCacheTest, VaryResponses) {
   // Responses will vary on accept.
-  const std::string RequestPath("some-resource");
+  const std::string RequestPath("/some-resource");
```

Comment:
s/RequestPath/request_path/. I realize the style guide says to use RequestPath for constants, but I'm pretty sure the intent is to use that for globals, not for local vars. Elsewhere in Envoy code we are not using that spelling for local const variables.